flow/designs/nangate45/ariane136/config_hier.mk

```diff
@@ -0,0 +1,15 @@
+include $(dir $(DESIGN_CONFIG))/config.mk
```

Comment:
There is no need for a config_hier.mk.  There were two configs during migration but they are being consolidated to one.  A new test doesn't need both.

Reply:
Ok. I'll merge both.